### DISCUSSION TITLE
GoTrue Postgres migrations

### DISCUSSION THREAD

**Discussion by vHanda** [supabase/supabase#255] [Questions] [answered]
Hi. I was planning to use GoTrue for a project of [mine](https://gitjournal.io), and saw that it only supported mysql, there is a [pull request](https://github.com/netlify/gotrue/pull/254) for postgres migrations, but that doesn't seem to be in your fork.

I would love to use postgres for my project, and I'd love if you could make your setup public. There is no mention for postgres in supabase/gotrue.

**Comment by kiwicopple** [comment, marked as answer]
Hey @vHanda - we actually use the code that's in the PR - we didn't add it to our fork because we were hoping that Netlify would merge it. We use the Namespace `auth` so that that the tables are set up in an `auth` schema.

If you can't get it working let us know and we can add instructions

**Reply by vHanda** [reply]
Hi. I've managed to get this to work on my local machine, however deploying it with docker totally fails for me. Both with mysql and postgres.

The problem I encounter is that `gotrue` doesn't seem to be able to connect to the database. The `DATABASE_URL` env variable is somehow being ignored with [`gobuffalo/pop`](https://github.com/gobuffalo/pop). It's value is passed to pop correctly, but when making the connection it seems to be ignored. I'm deep in its code and I just cannot figure it out. Is there anything special that you guys have done when creating the dockerfile?

**Reply by vHanda** [reply]
After hours and hours, I've figured out the problem.

I was using docker-compose and I had surrounded the `DATABASE_URL` variable with quotes, this resulted in the env variable in the container having quotes, which makes golang's `database/sql` silently fail and use the default settings which is to connect to `127.0.0.1`. 

I've filed a bug in in go - https://github.com/golang/go/issues/42380

**Reply by kiwicopple** [reply]
Interesting bug, I bet that was fun to find 😂 . Glad that you managed to find it and thanks for updating here - I'm sure this will be very useful to the community in the future
